revscores/scorers/linear_svc.py

```diff
@@ -74,7 +78,20 @@ def test(self, values_scores):
             },
             'auc': auc(fpr, tpr)
         }
+    
+    def _generate_stats(self, values):
+        columns = zip(*values)
+        
+        stats = tuple((mean(c), stdev(c)) for c in columns)
         
+        return stats
+    
+    def _scale_and_center(self, values, stats):
+        
+        for feature_values in values:
+            yield tuple((val-mean)/max(sd, 0.01)
```

Comment:
Is this division by 0.01 a good default when all the values are identicall (sd = 0)?